test/conftest.py

```diff
@@ -244,3 +245,10 @@ def pudl_datastore(pudl_settings_fixture, request):
             pudl_settings_fixture["pudl_in"]) / "data",
         gcs_cache_path=gcs_cache,
         sandbox=pudl_settings_fixture["sandbox"])
+
+
+@pytest.fixture(autouse=True, scope='session')
+def pandas_terminal_width():
+    """Set pandas displays for doctests."""
+    pd.options.display.width = 1000
+    pd.options.display.max_columns = 1000
```

Comment:
Should this still be here, or was it only for the doctests, which have now been removed?